### ISSUE TITLE
Move "Tell Your Story" on CNCF Case Studies into a box like https://kubernetes.io/case-studies/

### ISSUE THREAD

**Issue by oicheryl** [cncf/cncf.io#226]
I like the way that https://kubernetes.io/case-studies/ has a "Tell Your Story" box which links into a Google Form.

<img width="1474" alt="Screenshot 2020-08-14 at 14 16 18" src="https://user-images.githubusercontent.com/11319250/90253308-db7a9480-de38-11ea-991e-cf135ddeeccc.png">

<img width="1474" alt="Screenshot 2020-08-14 at 14 16 21" src="https://user-images.githubusercontent.com/11319250/90253320-df0e1b80-de38-11ea-9fcb-fddfdba8cd18.png">

Let's change https://www.cncf.io/newsroom/case-studies/ to follow suit (instead of linking to a long Github document which you then have to email.)

**Comment by thetwopct** [comment]
The Kubernetes page is great and the box at the end of that pile of logos suits that page. 

The new page on CNCF for case studies is here - https://dev-cncfci.pantheonsite.io/case-studies 

For CNCF, the case study box format is quite different (a lot larger, more details), and I am not sure an advert as part of the wall of case studies would work that well. There is lazy loading in place, so the user would need to scroll down quite a way to see the advert, and we need to work out how this would work with the search filters we have on that page, and the logic behind this (if no filters match, does it display?).

If the objective here is to promote to companies to tell their story, I can see a few alternatives:
 
1) Add it as a new paragraph to the intro text of that page
"Got a cloud native story you want to share? Contact us to arrange a case study on your company"

and/or

2) Add text + a button at the bottom to "Tell your story" (I've quickly added this to the page as an example)

3) Design something like we have on [Webinars](https://dev-cncfci.pantheonsite.io/webinars/develop-your-cloud-native-use-cases-at-the-edge-with-k3s/) to sit at the bottom of the page to encourage new submissions.

Use of a dedicated Google Form to handle submissions would be brilliant, and links can be updated to point to this when it's ready. (Also, the form could be hosted on CNCF website on a dedicated page - we can create it using Gravity Forms - and this may help track where submissions come from via Analytics)

**Comment by jkld11** [comment]
@thetwopct I like the idea of a button and text (linking to a form), but I think it would get lost at the bottom of that very long page. Is there any other option to move it to the top? Could be a button below the intro text.

**Comment by thetwopct** [comment]
For sure, easy to do. But I wonder if its a quite a powerful call to action to have there for every visitor to the page to have to see and scroll past? I wonder if a text link would be OK? Maybe something like [this](https://dev-cncfci.pantheonsite.io/case-studies/)

**Comment by jkld11** [comment]
Definitely a better location than it is now!

**Comment by cjyabraham** [comment]
I like the link at the top and think we should remove the CTA at the bottom of the page (it's annoying how you have to kind of chase it down the page as new case studies load).  Separately, I've split out a new issue for the form #231 for after we're live.

**Comment by thetwopct** [comment]
Updated the page. Once the form is ready we can update the link.
